xml/net_sync.xml

```diff
@@ -80,782 +74,364 @@
-  </para>
-
-  <sect2 xml:id="sec.net.sync.rsync.usage">
-   <title>Configuration and Operation</title>
-   <para>
-    rsync can be operated in two different modes. It can be used to archive or
-    copy data. To accomplish this, only a remote shell, like SSH, is required
-    on the target system. However, rsync can also be used as a daemon to
-    provide directories to the network.
+  <sect2 xml:id="sec.net.sync.rsync.server">
+   <title>Configuring and Using a Rsync Server</title>
+   <para> Rsync can be operated as a daemon (<systemitem class="daemon"
+     >rsyncd</systemitem>) listing on default port 873 for incoming
+    connections. This daemon can receive <quote>copying targets</quote>.
    </para>
    <para>
-    The basic mode of operation of rsync does not require any special
-    configuration. rsync directly allows mirroring complete directories onto
-    another system. As an example, the following command creates a backup of
-    the home directory of &exampleuser; on a backup server named
-    &exampleserver;:
+    The following procedure creates a Rsync server on
+    <systemitem>&wsI;</systemitem> with a <emphasis>backup</emphasis> target.
+    This target can be used to store your backups. To create a Rsync server,
+    do the following:
    </para>
-<screen>rsync -baz -e ssh<!--
-   --> /home/&exampleuser_plain;/ &exampleuser_plain;@&exampleserver;:backup</screen>
-   <para>
-    The following command is used to play the directory back:
-   </para>
-<screen>rsync -az -e ssh<!--
-   --> &exampleuser_plain;@&exampleserver;:backup /home/&exampleuser_plain;/</screen>
-   <para>
-    Up to this point, the handling does not differ much from that of a regular
-    copying tool, like scp.
-   </para>
-   <para>
-    rsync should be operated in <quote>rsync</quote> mode to make all its
-    features fully available. This is done by starting the rsyncd daemon on one
-    of the systems. Configure it in the file
-    <filename>/etc/rsyncd.conf</filename>. For example, to make the directory
-    <filename>/srv/ftp</filename> available with rsync, use the following
-    configuration:
-   </para>
-<screen>gid = nobody
-uid = nobody
-read only = true
-use chroot = no
-transfer logging = true
-log format = %h %o %f %l %b
-log file = /var/log/rsyncd.log
-
-[FTP]
-        path = /srv/ftp
-        comment = An Example</screen>
-   <para>
-    Then start rsyncd with <command>systemctl start rsyncd</command>. rsyncd
-    can also be started automatically during the boot process. Set this up by
-    activating this service in the &yast; <guimenu>Services Manager</guimenu>
-    or by manually entering the command:
-   </para>
-<screen>&prompt.root; systemctl enable rsyncd</screen>
-   <para>
-<!--<remark>taroth 2014-02-25: systemd - what about xinetd? is this
+   <procedure>
+    <title>Setting Up the Rsync Server</title>
+    <step>
+     <para>
+      Create a directory to store all your backup files.
```

Comment:
In Step 3 (configure the rsyncd daemon) you mention that we are working on `jupiter`.
Maybe here, on Step 1 it would be helpful too. Because it is not clear "where" we are.